x/ccv/consumer/types/keys.go

```diff
@@ -65,8 +65,9 @@ const (
 	// [DEPRECATED]
 	LastStandaloneHeightByteKey
 
-	// SmallestNonOptOutPowerByteKey is the byte that will store the smallest val power that cannot opt out
-	SmallestNonOptOutPowerByteKey
+	// NOTE: This key is deprecated, but left in place to avoid consumer state migrations
+	// [DEPRECATED]
+	DeprecatedSmallestNonOptOutPowerByteKey
```

Comment:
Cleanup the state in a migration.

Reply:
The state is 2 numbers (neutron and stride)
The migration can be skipped so we don't have to bump consensus versions .

Reply:
I think we already have to bump consensus versions for ICS 4.3. If we do, then let's just cleanup the state. Otherwise, just ignore.

Reply:
We already bump the consensus version of the `provider` module but **not** of the `consumer` module. Talked with @MSalopek about this and it would make things substantially more complicated in regards to ICS 5.0 because ICS 5.0 has already bumped the consumer to version 3.